release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -65,6 +65,16 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 * With this update, you can export your application in the ZIP file format to another project or cluster using the *Export application* option on the *+Add* page. 
 * With this update, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 
+[id="ocp-4-12-helm-page-improvements"]
+==== Helm page improvements
+
+In {product-title} 4.12, you can use the Helm page to:
+
+* Create Helm Releases and Repositories using the *Create* button.
+* Create, update, or delete a cluster-scoped or a namespace-scoped Helm chart repository.
+* View the list of the existing Helm chart repositories with their scope in the *Repositories* tab.
+* View the newly created Helm release in the *Helm Releases* tab.
```

Comment:
```suggestion
* View the newly created Helm release in the *Helm Releases* page.
```
Please use "page" for consistency with our other references to that term:
```
Results for "*Helm Releases* page" in "openshift-docs/*.adoc"

/home/rdlugyhe/openshift-docs/modules/odc-rolling-back-helm-release.adoc:
  13:  . In the *Helm Releases* page, click on the chart to see the details and resources for that release.


/home/rdlugyhe/openshift-docs/modules/odc-upgrading-helm-release.adoc:
  11:  then click *Upgrade* to create another Helm release. The *Helm Releases* page displays the two revisions.
```

Reply:
Again, as I stated in the earlier comment, I was asked by the QE to change it to tab instead of page. Adding the PR link here (https://github.com/openshift/console/pull/11844) and requesting for further guidance from you for accurate usage of the associated term in this instance.

Reply:
The QE/SME, though well-intentioned, is using "tab" as a proxy for "page".  They are unaware that we distinguish between a tab, which is a navigation element, and a page, which displays content.

Reply:
Fixed.